apps/frontend/src/features/leaderboard/components/LeaderboardStateMessage.tsx

```
@@ -30,10 +30,12 @@ export const LeaderboardStateMessage = ({ state, message }: LeaderboardStateMess
 
   return (
     <>
-      <div css={stateCardStyle(theme)}>{displayMessage}</div>
+      <div css={stateCardStyle(theme)} role="status" aria-live="polite">
+        {displayMessage}
+      </div>
 
       {showSkeleton && (
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)